common/shell/BUILD

```diff
@@ -0,0 +1,31 @@
+#
```

Comment:
Moved from `platform.jvm` to `common` as it is now used by both `assemble_jvm_platform` and `deploy_npm`